test/setupTests.ts

```diff
@@ -48,23 +52,21 @@ try {
   });
 } catch {}
 
-const [firstPath, ...rest] = await Promise.all(
+const [, ...rest] = await Promise.all(
```

Comment:
```suggestion
const templates = await Promise.all(
```

Reply:
but that's completely different. in my case I use everything except the first one

Reply:
sry, didn't notice comma
i meant `const [, templates] = await Promise.all(`
because `rest` variable name is not very self-descriptive